README.md

```diff
@@ -165,6 +165,20 @@ xe host-call-plugin host-uuid=<uuid>  plugin=smartctl.py fn=health
 {"/dev/sdf": "PASSED", "/dev/sdg": "PASSED", "/dev/sdd": "PASSED", "/dev/sde": "PASSED", "/dev/sdb": "PASSED", "/dev/sdc": "PASSED", "/dev/sda": "PASSED"}
 ```
 
+## IPMI-sensors parser
+A xapi plugin to get information about the host via the BMC
+### `get_info`:
+```
+xe host-call-plugin host-uuid=<uuid>  plugin=2crsi-sensors.py fn="get_info"
```

Comment:
IMHO, it would be better and more robust to have a generic IPMI plugin, no need to have logic both in XO and the plugin that will require synchronization on updates.

Possibly accepting a list of fields to return if we want to fetch only a subset.